PKHeX/MainWindow/Main.cs

```diff
@@ -4304,7 +4315,21 @@ public static void Reset()
                 return slotSource == form || slotDestination == form; // form already updated?
             }
         }
+        public static bool IsLinkedSlot(int slot, int box) {
+            for (int i = 0; i < 72; i++) {
+                int lslot = SAV.getData(19652 + i, 1)[0];
+                i++;
+                int lbox = SAV.getData(19652 + i, 1)[0];
+                if (lbox == box) {
+                    if (slot == lslot) {
+                        Util.Alert("Failed to modify slot "+ ++slot +" of box "+ ++box +" because is linked to a team. Take the Pokemon out of it in-game before trying again.");
```

Comment:
The function's name implies it simply determines whether or not a slot is linked.  As such, I don't think Util.Alert should be used here.